modules/indexing/src/main/java/org/apache/ignite/internal/mxbean/SqlQueryMXBeanImpl.java

```diff
@@ -72,4 +72,34 @@ public SqlQueryMXBeanImpl(GridKernalContext ctx) {
     @Override public void setResultSetSizeThresholdMultiplier(int rsSizeThresholdMultiplier) {
         h2idx.longRunningQueries().setResultSetSizeThresholdMultiplier(rsSizeThresholdMultiplier);
     }
+
+    /** {@inheritDoc} */
+    @Override public String getSqlGlobalMemoryQuota() {
+        return h2idx.memoryManager().getGlobalQuota();
+    }
+
+    /** {@inheritDoc} */
+    @Override public void setSqlGlobalMemoryQuota(String size) {
+        h2idx.memoryManager().setGlobalQuota(size);
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getSqlQueryMemoryQuota() {
+        return h2idx.memoryManager().getQryQuotaStr();
+    }
+
+    /** {@inheritDoc} */
+    @Override public void setSqlQueryMemoryQuota(String size) {
+        h2idx.memoryManager().setQueryQuota(size);
+    }
```

Comment:
in one place we use abbreviation Query for second one it is Query - need to have single style.

Reply:
Done